public/app/core/navigation/GrafanaRouteError.tsx

```diff
@@ -51,5 +51,8 @@ export function GrafanaRouteError({ error, errorInfo }: Props) {
 }
 
 const getStyles = stylesFactory(() => {
-  return css``;
+  return css`
+    width: 500px;
```

Comment:
Hi @torkelo, should we change this to `max-width` to avoid responsive problems? What do you think? 🤔